active/radiobuttons/RadioButtonsAPI.md

```diff
@@ -0,0 +1,182 @@
+```
+
+![alt text](multicolumns.png)
+
+# API Details
+
+## RadioButtons
+
+```
+namespace Microsoft.UI.Xaml.Controls
+{
+[webhosthidden]
+[contentproperty("Items")]
+unsealed runtimeclass RadioButtons : Windows.UI.Xaml.Controls.Control
+{
+    RadioButtons();
+
+    Object ItemsSource;
+    Windows.Foundation.Collections.IVector<Object> Items{ get; };
+    Windows.UI.Xaml.DataTemplate ItemTemplate;
+
+    Windows.UI.Xaml.DependencyObject ContainerFromItem(Object item);
+    Windows.UI.Xaml.DependencyObject ContainerFromIndex(Int32 index);
+
+    Int32 SelectedIndex;
+    Object SelectedItem;
+    event Windows.UI.Xaml.Controls.SelectionChangedEventHandler SelectionChanged;
+
+    Int32 MaxColumns;
+    Object Header;
+
+    static Windows.UI.Xaml.DependencyProperty ItemsSourceProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty ItemsProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty ItemTemplateProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty SelectedIndexProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty SelectedItemProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty MaxColumnsProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty HeaderProperty{ get; };
+}
+}
+```
+
+## RadioButtons Automation Peer
+
+```
+namespace Microsoft.UI.Xaml.Automation.Peers
+{
+[webhosthidden]
+unsealed runtimeclass RadioButtonsListViewItemAutomationPeer : Windows.UI.Xaml.Automation.Peers.ListViewItemAutomationPeer
+{
+    RadioButtonsListViewItemAutomationPeer(RadioButtonsListViewItem owner);
+}
+}
+```
+
+## RadioButtons Primitives
+
+```
+namespace Microsoft.UI.Xaml.Controls.Primitives
+{
+[webhosthidden]
+unsealed runtimeclass RadioButtonsListView : Windows.UI.Xaml.Controls.ListView
+{
+    RadioButtonsListView();
+}
+
+[webhosthidden]
+[default_interface]
+unsealed runtimeclass RadioButtonsListViewItem : Windows.UI.Xaml.Controls.ListViewItem
+{
+    RadioButtonsListViewItem();
+}
+}
+```
+
+# API Notes
+
+| Name | Description |
+|:-:|:--|
+| Header | Places a text label above the group of RadioButton elements and is exposed to the UIA tree via the DataContext property on the RadioButtons control's ContentPresenter. |
+| HeaderTemplate | The template that specifies the visualization of the header object |
+| SelectedItem | Gets or sets the index of the selected item. Selection is denoted by that item's RadioButton icon being checked. |
+| SelectedIndex | Gets or sets the selected item.|
+| Items | Gets or sets an object source used to generate the content of the ItemsControl. All items placed within the RadioButtons control will get the content of a generated RadioButton. |
+| ItemsSource | Gets or sets an object source used to generate the content of the ItemsControl. |
+| ItemTemplate | The template set on any generated RadioButton elements within the RadioButtons control. If the root of the ItemTemplate is a RadioButton it will be used as the generated container. |
```

Comment:
Perhaps an example of using ItemTemplate would be helpful?

Reply:
I believe I added this. Resolving.